great-idea/index.html

```diff
@@ -14,10 +14,120 @@
   <![endif]-->
 </head>
 
-  <body>
-
-    <!-- Copy and paste your HTML from the first UI project here -->
+<body>
+    <div class="header">
+        <nav>
+            <a href="services.html">Services</a>
+            <a href="#">Product</a>
+            <a href="#">Vision</a>
+            <a href="#">Features</a>
+            <a href="#">About</a>
+            <a href="#">Contact</a>
+            <P><a href="index.html"><img class="logo" src="img/logo.png" alt="Great Idea! Company logo."></a></P>
```

Comment:
was there a reason for the p tag?

Reply:
It was not working with a for some reason, but I cleaned it up!